api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -3294,4 +3294,10 @@ public void getPatientIdentifiersByPatientProgram_shouldGetPatientsByIdentifierB
 		assertEquals(patientIdentifier.iterator().next().getIdentifier(), "XXXCCCAAA11");
 	}
 
+	@Test
+	public void getIdentifierValidator_shouldThrowPatientIdentifierExceptionWhenClassNotFound() throws Exception {
+		PatientIdentifierException patientIdentifierException = assertThrows(PatientIdentifierException.class, () -> patientService.getIdentifierValidator("com.example.InvalidIdentifierValidator"));
```

Comment:
Can you also add a test for should not throw exception when class is found?

Reply:
@dkayiwa It seems in the `patientService`, [this test](https://github.com/openmrs/openmrs-core/blob/ddf8b1bf17a0ad9cf488888c0713f15301a075be/api/src/test/java/org/openmrs/api/PatientServiceTest.java#L1199) has already added  , So it checks whether the `getIdentifierValidator` method returns the expected IdentifierValidator class when found.